src/main/java/com/dev/ForkJoinMain.java

```diff
@@ -0,0 +1,30 @@
+package com.dev;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.Future;
+
+public class ForkJoinMain {
```

Comment:
let's create a separates classes for this and use only one main method where we will  call both of these variations of calculating the sum.